services/venice-server/build.gradle

```diff
@@ -63,6 +70,25 @@ jar {
   }
 }
 
+def grpcVersion = '1.54.1'
+def protobufVersion = '3.21.7'
+
+protobuf {
```

Comment:
let's move this to the parent `build.gradle` and then make this plugin and task available via `subprojects {..}`

Reply:
done, moved dependencies and tasks to parent `build.gradle` file.